solutions/standard/outputs.tf

```diff
@@ -59,3 +59,13 @@ output "certificate_base64" {
   value       = module.redis.certificate_base64
   sensitive   = true
 }
+
+output "service_credential_secrets" {
+  description = "Service credential secrets"
+  value       = length(local.service_credential_secrets) > 0 ? module.secrets_manager_service_credentials[0].secrets : null
+}
+
+output "service_credential_secret_groups" {
```

Comment:
This is not consistent with elasticsearch which does not output this value. It should follow the naming of the secrets_manager_secrets/service_credential_secrets output above.